pkg/controller/kubelet-config/helpers.go

```diff
@@ -17,13 +18,17 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/kubelet/config/v1beta1"
 	kubeletconfigv1beta1 "k8s.io/kubelet/config/v1beta1"
 
 	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
 	ctrlcommon "github.com/openshift/machine-config-operator/pkg/controller/common"
 	mcfgclientset "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned"
 )
 
+// MaxMCNameSuffix is the maximum value of the machineconfig name suffix
+const MaxMCNameSuffix int = 9
```

Comment:
+1 creating the const, but also you'd need to use it in the other occurrence of the non-const 9 which is below. See: https://github.com/openshift/machine-config-operator/blob/2ec816a4aa741821e664fa512ab02f465926c0ab/pkg/controller/kubelet-config/helpers.go#L209

Reply:
basically i asked for the const so these two usages wouldnt skew =)

Reply:
Fixed https://github.com/openshift/machine-config-operator/pull/2547/files#diff-4a2d0a4442e10d889a927322ad7311c67b22b259622067b52cbda7d2ece3ebefR210